source/javascript-api-reference/index.html.md.erb

```diff
@@ -5,32 +5,19 @@ weight: 80
 
 # JavaScript API Reference
 
-Some of our components can receive configuration options when you create an
-instance in JavaScript. This page lists the available options for these
-components. You can find further information on how to [configure these options
-in our guidance](../configure-components/).
+Some of our components can be [passed JavaScript configuration](../configure-components/#passing-javascript-configuration) objects as:
```

Comment:
I think the link points a bit too high in the heading of the other page. Once you're there you have to figure out which sub-section applies. How about pointing to the specific sections with:

```suggestion
Some of our components can be passed JavaScript configuration objects. You can do it both:
- [when configuring individual component instances](../configure-components/#with-configuration-object)
- [when configuring all components using the `initAll` function](../configure-components/#configure-all-components-using-the-initall-function)
```

That would also save us talking about the precision of which argument does what, where "first argument" doesn't quite represent where the configuration for a given component goes (it is the first argument, but under a key for the element name).